run_integration_tests.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+# Install kw before running tests to ensure we are testing the most recent version
+./setup.sh -i -v
```

Comment:
I don't think `-v` is a valid option

Reply:
wow, you are right, somehow all this time I have been using -v thinking I was running in verbose mode, thank you for catching that